CONTRIBUTING.md

```diff
@@ -97,14 +79,6 @@ You are expected to install your own workflow the way you like it. For example,
 some developers like to auto-format code "on save", others before they commit or
 push their changes. That's why this repository does not auto-install Git hooks.
 
-If you want to make sure you don't forget to update the changelog before you
-push code, here's an example to install a Git hook for that:
-
-```bash
-echo "rush change --verify" > .git/hooks/pre-push
-chmod +x .git/hooks/pre-push
-```
-
```

Comment:
changesets will take care of this, it's awesome. If you forget to run `pnpm changesets` you will be notified through a comment in the PR and an email.